src/main/java/it/pn/frontend/e2e/pages/destinatario/personaFisica/NotifichePFPage.java

```diff
@@ -127,12 +129,9 @@ public boolean getListData() {
     public boolean verificaEsistenzaEPassaggioPagina() {
         this.js().executeScript("window.scrollBy(0,document.body.scrollHeight)");
         try {
-            By numeroButtonBy = By.id("page2");
-            this.getWebDriverWait(20).withMessage("bottone pagina 2 non è visibile").until(ExpectedConditions.visibilityOfElementLocated(numeroButtonBy));
+            this.getWebDriverWait(20).withMessage("bottone pagina 2 non è visibile").until(ExpectedConditions.visibilityOf(paginaSeconda));
             logger.info("Bottone pagina 2 trovato");
-
-            WebElement numeroPagina = this.element(numeroButtonBy);
-            numeroPagina.click();
+            paginaSeconda.click();
```

Comment:
Check `clickPagina()` in  `PiattaformaNotifichePage.java` there is already a method that should work for every platform(pa,pf,pg)